docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,707 @@
+
+Infra/REST API::
+* Copy HTTP headers to `ThreadContext` strictly {es-pull}45945[#45945]
+
+Machine Learning::
+* Remove the `MachineLearningClient` {es-pull}43108[#43108]
+
+Mapping::
+* Remove type filter from `GetMappings` API {es-pull}47364[#47364] (issue: {es-issue}41059[#41059])
+* Remove `type` parameter from `PutMappingRequest.buildFromSimplifiedDef()` {es-pull}50844[#50844] (issue: {es-issue}41059[#41059])
+* Remove unused parameter from `MetadataFieldMapper.TypeParser#getDefault()` {es-pull}51219[#51219]
+* Remove `type` parameter from `CIR.mapping(type, object...)` {es-pull}50739[#50739] (issue: {es-issue}41059[#41059])
+
+Search::
+* Removes types from `SearchRequest` and `QueryShardContext` {es-pull}42112[#42112]
+
+Snapshot/Restore::
+* Remove deprecated repository methods {es-pull}42359[#42359] (issue: {es-issue}42213[#42213])
+
+
+[[deprecation-8.0.0]]
+[float]
+=== Deprecations
+
+Authentication::
+* Deprecate setup-passwords tool {es-pull}76902[#76902]
+
+CRUD::
+* Remove `indices_segments` 'verbose' parameter {es-pull}78451[#78451] (issue: {es-issue}75955[#75955])
+
+Engine::
+* Deprecate setting `max_merge_at_once_explicit` {es-pull}80574[#80574]
+
+Machine Learning::
+* Deprecate `estimated_heap_memory_usage_bytes` and replace with `model_size_bytes` {es-pull}80554[#80554]
+
+Monitoring::
+* Add deprecation info API entries for deprecated monitoring settings {es-pull}78799[#78799]
+* Automatically install monitoring templates at plugin initialization {es-pull}78350[#78350]
+* Remove Monitoring ingest pipelines {es-pull}77459[#77459] (issue: {es-issue}50770[#50770])
+
+Search::
+* Configure `IndexSearcher.maxClauseCount()` based on node characteristics {es-pull}81525[#81525] (issue: {es-issue}46433[#46433])
+
+Transform::
+* Improve transform deprecation messages {es-pull}81847[#81847] (issues: {es-issue}81521[#81521], {es-issue}81523[#81523])
+
+[[feature-8.0.0]]
+[float]
+=== New features
+
+Security::
+* Auto-configure TLS for new nodes of new clusters {es-pull}77231[#77231] (issues: {es-issue}75144[#75144], {es-issue}75704[#75704])
+
+Snapshot/Restore::
+* Support IAM roles for Kubernetes service accounts {es-pull}81255[#81255] (issue: {es-issue}52625[#52625])
+
+Watcher::
+* Use `startsWith` rather than exact matches for Watcher history template names {es-pull}82396[#82396]
+
+
+[[enhancement-8.0.0]]
+[float]
+=== Enhancements
+
+Analysis::
+* Move `reload_analyzers` endpoint to x-pack {es-pull}43559[#43559]
+
+Authentication::
+* Reset elastic password CLI tool {es-pull}74892[#74892] (issues: {es-issue}70113[#70113], {es-issue}74890[#74890])
+* Autogenerate and print elastic password on startup {es-pull}77291[#77291]
+* Enroll Kibana API uses Service Accounts {es-pull}76370[#76370]
+* Add `reset-kibana-system-user` tool {es-pull}77322[#77322]
+* New CLI tool to reset password for built-in users {es-pull}79709[#79709]
+* Upgrade to UnboundID LDAP SDK v6.0.2 {es-pull}79332[#79332]
```

Comment:
Should this go under "Upgrades"?

Reply:
Moved in https://github.com/elastic/elasticsearch/pull/83689/commits/d59e494ee857c2b2e50389997d70bb2b56353a90